### PR TITLE
Clarify linked document and fix pdf link

### PR DESCRIPTION
**Pull request by Ainali**
There are many PDF documents on the page, so it might be helpful to specify which one to look for. Also, the direct link to the PDF was going to the same webpage.

-----
[View rendered activities/value-and-impact/policy-documents.md](https://github.com/publiccodenet/about/blob/ja-fix-pdf-link/activities/value-and-impact/policy-documents.md)

### CODE REVIEW

**Comment by ElenaFdR** [comment]
Here's the logic for linking to the page, rather than the PDF: https://github.com/publiccodenet/about/pull/867#pullrequestreview-592406568

**Comment by Ainali** [comment]
> Here's the logic for linking to the page, rather than the PDF: [#867 (review)](https://github.com/publiccodenet/about/pull/867#pullrequestreview-592406568)

But it has two links to the same page, one to the page, and one that claims to be a direct link to the PDF (but isn't, that's my fix). If we don't want the link to the PDF, the `[PDF]` should be removed completely.

**Comment by Ainali** [comment]
> > Here's the logic for linking to the page, rather than the PDF: [#867 (review)](https://github.com/publiccodenet/about/pull/867#pullrequestreview-592406568)
> 
> But this entry has two links to the same page, one to the page, and one that claims to be a direct link to the PDF (but isn't, that's my fix). If we don't want the link to the PDF, the `[PDF]` should be removed completely.

`to achieve the goals set out in the [A Digital City Agenda (Agenda Digitale Stad)](https://www.amsterdam.nl/wonen-leefomgeving/innovatie/de-digitale-stad/) [[PDF]](https://www.amsterdam.nl/wonen-leefomgeving/innovatie/de-digitale-stad/) 
`

This is the redundancy I want to fix. Two links, labelled differently, but that goes to the same place. In my opinion, if the second one (labelled PDF) should still be there, it should link somewhere else. And why not to the PDF directly when the first one links to the page hosting it?

**Comment by ElenaFdR** [comment]
@ainali, I agree with your proposal, but don't think the way we currently present PDFs is distinct enough for someone to notice that the very long hyperlink for the report name is a different from the link to [PDF]. I had a quick look yesterday for conventions for presenting PDFs vs holding pages for PDFs when there isn't an HTML version of the PDF, but haven't found one yet that looks like a nice one for us to borrow.

**Comment by ElenaFdR** [comment]
@Ainali I like it! It's subtle, but let's try it.

**Comment by ElenaFdR** [comment]
@Ainali What do you think of my most recent changes?

**Comment by Ainali** [comment]
> @Ainali What do you think of my most recent changes?

I like it, very clear!

File: activities/value-and-impact/policy-documents.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ explains: where we are mentioned in public policy documents
...
+* Amsterdam municipality: Mid-term report of the A Digital City Agenda (Agenda Digitale Stad [[PDF]](https://amsterdam.raadsinformatie.nl/document/9725293/1/09012f9783374971) (in Dutch)
 
 ## 2019
 
-* Amsterdam municipality: The [Sourcing and Open Source Strategy Amsterdam [PDF]](https://amsterdam.raadsinformatie.nl/document/7880446/1/09012f9782a30a08) is one of the municipal strategies to achieve the goals set out in the [A Digital City Agenda (Agenda Digitale Stad)](https://www.amsterdam.nl/wonen-leefomgeving/innovatie/de-digitale-stad/) [[PDF]](https://www.amsterdam.nl/wonen-leefomgeving/innovatie/de-digitale-stad/) (in Dutch)
+* Amsterdam municipality: The Sourcing and Open Source Strategy Amsterdam [[PDF]](https://amsterdam.raadsinformatie.nl/document/7880446/1/09012f9782a30a08) is one of the municipal strategies to achieve the goals set out in the [A Digital City Agenda]((https://www.amsterdam.nl/wonen-leefomgeving/innovatie/de-digitale-stad/))
```
Comment (Ainali) [inline activities/value-and-impact/policy-documents.md:20]:
```suggestion
* Amsterdam municipality: The Sourcing and Open Source Strategy Amsterdam [[PDF]](https://amsterdam.raadsinformatie.nl/document/7880446/1/09012f9782a30a08) is one of the municipal strategies to achieve the goals set out in the [A Digital City Agenda](https://www.amsterdam.nl/wonen-leefomgeving/innovatie/de-digitale-stad/)
```

File: activities/value-and-impact/policy-documents.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ explains: where we are mentioned in public policy documents
 
 ## 2021
 
-* State of California: [Vision 2023 - California Technology Strategic Plan](https://vision2023.cdt.ca.gov/) [[PDF]](https://vision2023.cdt.ca.gov/pdf/Vision-2023-California-Technology-Strategic-Plan.pdf)
+* State of California: Vision 2023 - California Technology Strategic Plan [[web]]((https://vision2023.cdt.ca.gov/)) [[PDF]](https://vision2023.cdt.ca.gov/pdf/Vision-2023-California-Technology-Strategic-Plan.pdf)
```
Comment (Ainali) [inline activities/value-and-impact/policy-documents.md:10]:
```suggestion
* State of California: Vision 2023 - California Technology Strategic Plan [[web]](https://vision2023.cdt.ca.gov/) [[PDF]](https://vision2023.cdt.ca.gov/pdf/Vision-2023-California-Technology-Strategic-Plan.pdf)
```

**Review by ericherman — DISMISSED** [review]

**Review by bvhme — APPROVED** [review]
Looks good

**Pull request merged**